src/GUI/EFCorePowerTools/Handlers/ReverseEngineerHandler.cs

```diff
@@ -72,8 +75,10 @@ public async void ReverseEngineerCodeFirst(Project project)
                     return;
 
                 var useEFCore5 = pickDataSourceResult.Payload.IncludeViews;
+                var filterSchemas = pickDataSourceResult.Payload.FilterSchemas;
+                var schemas = filterSchemas ? pickDataSourceResult.Payload.Schemas : null;
 
-                _package.Dte2.StatusBar.Text = "Loading schema information...";
+                _package.Dte2.StatusBar.Text = "Loading schema(s) information...";
```

Comment:
Please revert this line

Reply:
Actually change to: "Loading table list..."

Reply:
Fixed